azurerm/resource_arm_api_management.go

```diff
@@ -562,8 +585,21 @@ func resourceArmApiManagementServiceRead(d *schema.ResourceData, meta interface{
 		}
 	}
 
-	if err := d.Set("sku", flattenApiManagementServiceSku(resp.Sku)); err != nil {
-		return fmt.Errorf("Error setting `sku`: %+v", err)
+	if skuName == "" {
```

Comment:
As mentioned in other PR, we can just set both and simplify this logic

Reply:
I now set both and removed the logic...